plumbing/transport/file/server.go

```diff
@@ -0,0 +1,82 @@
+package file
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/src-d/go-git.v4/plumbing/transport"
+	"gopkg.in/src-d/go-git.v4/plumbing/transport/server"
+	"gopkg.in/src-d/go-git.v4/storage/filesystem"
+	"gopkg.in/src-d/go-git.v4/utils/ioutil"
+
+	osfs "srcd.works/go-billy.v1/os"
+)
+
+var DefaultServer = NewServer(server.DefaultHandler)
+
+type Server struct {
```

Comment:
having the server at this level don't allow to you to having a server of multiple repositories, I was thinking on having a server package at porcelain level